apps/credential-showcase-api-server/src/database/repositories/__tests__/persona.repository.test.ts

```diff
@@ -0,0 +1,148 @@
+import 'reflect-metadata';
+
+import { drizzle } from 'drizzle-orm/pglite'
+import { migrate } from 'drizzle-orm/node-postgres/migrator'
+import { NewPersona } from '../../../types'
+import PersonaRepository from '../PersonaRepository'
+import { Container } from 'typedi'
+import { DatabaseService } from '../../../services/DatabaseService'
+import { PGlite } from '@electric-sql/pglite'
+import { NodePgDatabase } from 'drizzle-orm/node-postgres'
+import * as schema from '../../schema'
+
+describe('Database persona repository tests', (): void => {
```

Comment:
we need to check the assets on full objects not only id's, as this is in the api spec